### PR TITLE
feeluown: 3.7.3+git20210211

### PR DESCRIPTION
**Pull request by Fearyncess**
Topic Description
-----------------
- Update `FeelUOwn` to 3.7.3+git20210211.
- Update `qasync` to 0.13.0.
- Update `feeluown-netease` to 0.5.
- Add `pydantic`
- Drop `feeluown-download`

Package(s) Affected
-------------------
- Add: `pydantic`
- Update: `qasync`
- Update: `feeluown`
- Update: `feeluown-netease`
- Drop :`feeluown-download`

Security Update?
----------------
No 

Build Order
-----------
- `pydantic`, `qasync` => `feeluown` => `feeluown-netease`

Architectural Progress
----------------------
 - [x] Architecture-independent `noarch`

----

After the pull request is merged, all package(s) affected must be rebuilt against the `stable` Git tree and environment (only `stable` repository should be enabled in `sources.list`). This section marks the progress above.

Please, make sure the list of architectures below matches the ones above.

Post-Merge Architectural Progress
---------------------------------
 - [ ] Architecture-independent `noarch`

### CODE REVIEW

**Comment by Fearyncess** [comment]
image
LGTM

File: extra-multimedia/feeluown/autobuild/beyond [hunk 1]
```diff
@@ -14,5 +14,5 @@ EOF
 abinfo "Removing feeluown-genicon ... "
 rm -fv "$PKGDIR"/usr/bin/feeluown-genicon
 
-install -Dm644 "$SRCDIR"/feeluown/feeluown.png \
+install -Dm644 "$SRCDIR"/feeluown/icons/feeluown.png \
```
Comment (MingcongBai) [inline extra-multimedia/feeluown/autobuild/beyond]: Please add verbosity here, and declare progress with `abinfo`.

File: extra-multimedia/feeluown/spec [hunk 2]
```diff
@@ -1,3 +1,4 @@
-VER=3.5.3
-SRCTBL="https://pypi.org/packages/source/f/feeluown/feeluown-3.5.3.tar.gz"
-CHKSUM="sha256::b06d3ee831e3accef9a52fa3e9ac100b6c63fe76caa58f5f3de53356823757f7"
+VER=3.7.3+git20210211
+SRCS="git::commit=60611a7163efc20b487593e65fbf2789a58f9004::https://github.com/feeluown/FeelUOwn"
+CHKSUMS="SKIP"
+#SRCS="https://pypi.org/packages/source/f/feeluown/feeluown-${VER}.tar.gz"
```
Comment (MingcongBai) [inline extra-multimedia/feeluown/spec]: Drop unneeded line.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: extra-multimedia/feeluown/autobuild/beyond [hunk 3]
```diff
@@ -1,18 +1,5 @@
...
-Categories=AudioVideo;Audio;Player;Qt;
-Terminal=false
-StartupNotify=true
-EOF
-
 abinfo "Removing feeluown-genicon ... "
 rm -fv "$PKGDIR"/usr/bin/feeluown-genicon
 
-install -Dm644 "$SRCDIR"/feeluown/feeluown.png \
+install -v -Dm644 "$SRCDIR"/feeluown/icons/feeluown.png \
```
Comment (MingcongBai) [inline extra-multimedia/feeluown/autobuild/beyond:5]: Please add `abinfo` to note progress.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**